tutorons/python/explain.py

```diff
@@ -0,0 +1,44 @@
+#! /usr/bin/env python
+# encoding: utf-8
+
+from __future__ import unicode_literals
+import logging
+from bs4 import BeautifulSoup
+import collections
+
+logging.basicConfig(level=logging.INFO, format="%(message)s")
+
+fcns = open('tutorons/python/fcns.txt', 'r').read()
```

Comment:
In general, I like to use the construct

    with open(<filename>, 'r') as file_:
        varname = file_.read()

The nice thing about this is that it will automatically close the file after it is read.